src/scripts/components/structures/Card.js

```diff
@@ -64,7 +66,7 @@ export const CardWrapper = styled.div`
 
 const Main = styled.div`
   background: ${props => props.theme.colors.MainCard.background}
-    ${props => (props.nobackground ? null : "url('assets/svg/card-bg.svg')")}
+    ${props => (props.nobackground ? null : `url(${cardBackgroundSvgDataUrl})`)}
```

Comment:
@jrgarou can you help explain why we need this svg background at all? It seems to just be a solid color background anyways

Reply:
After a lot of changes, there're few cards with the background - or no one maybe.